tox.ini

```diff
@@ -1,33 +1,19 @@
 [tox]
 envlist =
-    py27,
-    py33,
-    py34,
-    py35,
-    py36,
-    pypy,
-    pypy3,
-    coverage-report,
+    py27,pypy,py34,py35,py36,pypy3,coverage
 
 [testenv]
 commands =
-    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
+    zope-testrunner --test-path=src []
 deps =
     .[test]
-    zope.testrunner
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
 
-[testenv:coverage-report]
-basepython = python2.7
-deps = coverage
-setenv =
-    COVERAGE_FILE=.coverage
-skip_install = true
+
+[testenv:coverage]
```

Comment:
Okay, understood.
Are there any future plans to run the coverage tests on a Python 3 version or will we do this when dropping Python 2 support?